src/components/Recipe.test.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { rest } from "msw";
```

Comment:
I think it is much easy to just mock the method calls with jest, you're going down deep to the implementation details mocking the server

https://jestjs.io/docs/en/mock-functions